javaslang/src/main/java/javaslang/collection/Vector.java

```diff
@@ -22,15 +24,19 @@
  * @author Ruslan Sennov
  * @since 2.0.0
  */
-public final class Vector<T> implements Kind1<Vector<?>, T>, IndexedSeq<T>, Serializable {
-
+@SuppressWarnings({"SuspiciousArrayCast", "unchecked"})
+public class Vector<T> implements Kind1<Vector<?>, T>, IndexedSeq<T> {
```

Comment:
> It was redundant, already defined in interface λ<R> extends Serializable.
> I could still leave it there if you think it's more readable, it just makes no difference to the compiler.

Thank you, it's OK for me.
And second question: why `Vector` is not final now? I don't remember exactly, but this keyword very important here... /cc @danieldietrich